### ISSUE TITLE
Replace AVA messaging with QuickSubmit text and link on 2 static pages - and audit for other opportunities

### ISSUE THREAD

**Issue by bethpottsVADEPO** [department-of-veterans-affairs/va.gov-team#61267]
## Issue Description
We want to provide Veterans with clear messaging and links so they know which of these tools to use for questions/help and which to use for uploading benefit-related supporting documents (not education benefit applications themselves--except for the licensing and certification use case outlined under tasks in this ticket): **Ask VA (AVA) vs QuickSubmit**. 

We know of 2 existing static benefit pages where we need to update text and links accordingly, and we need to audit our site for other opportunities to improve/correct messaging within the Education benefit hub.

**Background:** The problem Education Service is looking to solve is that Veterans are uploading **supporting documents (DD214, etc.)** through Ask VA, and they would like Veterans to use Quick Submit instead. There is the ability to upload documents through AVA in case that's helpful when VA is answering a question a Veteran may have. But they want to make the distinction between that "getting help" scenario vs uploading documents to support an education benefits claim.

On a call with regional processing office contacts, they specifically called out that they do want the licensing and certification app uploaded through QS. When we go to them with these screenshots, we can ask if there are any other instances other than L&C where they want Veterans to use QS.

We may also need guidance on how to use QuickSubmit. Here's the language EDU has provided on how to use QuickSubmit:

If you are only submitting documents for VA Education claims, you should submit directly to Centralized Mail Portal via QuickSubmit for better customer service. It’s quick, easy, and ensures the documents get into file quickly. To log in to QuickSubmit, a VA PIV Card or an ID.me account is required. Access QuickSubmit via AccessVA at https://eauth.va.gov/accessva/?cspSelectFor=quicksubmit
• Users must complete First Name, Last Name, File Number/SSN and Zip Code
• Select Education for both “Organization” and “Benefit Claim Type”
• “Emergent Indicator” must be left blank
Once the fields are completed, you may select the PDF file to attach, or drag and drop the file into the appropriate area. After documents are attached, select Submit.

QuickSubmit link to use: https://eauth.va.gov/accessva/?cspSelectFor=quicksubmit

---
## Tasks
- [x] Update the education [How to apply](https://www.va.gov/education/how-to-apply/) page under the header "What happens after I apply?" Change the messaging there to: "If we've asked you for documents, you can upload them using QuickSubmit on AccessVA." </br>
**Link text:** Upload supporting documents using QuickSubmit on AccessVA</br>
**Destination:** https://eauth.va.gov/accessva/?cspSelectFor=quicksubmit
- [ ] Update page about licensing and certification called [Get paid back for test fees](https://www.va.gov/education/about-gi-bill-benefits/how-to-use-benefits/test-fees/) under the header "How do I get this benefit?" The first accordion in that section has info about licensing and certification. Instead of the text and link driving users to Ask VA, change it to QuickSubmit in a similar way as the "How to apply" page approach.
- [ ] Find other opportunities to correct messaging and links in the education hub that currently drive users to Ask VA instead of QuickSubmit.

## Acceptance Criteria
- [ ] Unauth pages in the education hub are correctly sending users to Ask VA when they need help or have a question, and to QuickSubmit instead when they need to upload documents to support their education benefit claim.
---
## How to configure this issue
- [ ] **Attached to a Milestone** (when will this be completed?)
- [ ] **Attached to an Epic** (what body of work is this a part of?)
- [ ] **Labeled with Team** (`product support`, `analytics-insights`, `operations`, `service-design`, `Console-Services`, `tools-fe`)
- [ ] **Labeled with Practice Area** (`backend`, `frontend`, `devops`, `design`, `research`, `product`, `ia`, `qa`, `analytics`, `contact center`, `research`, `accessibility`, `content`)
- [ ] **Labeled with Type** (`bug`, `request`, `discovery`, `documentation`, etc.)

**Comment by bethpottsVADEPO** [comment]
@katherine-fung The approach to take for this work is to get the updates to the 2 pages already identified done first, since that's an easy lift. Get screen shots off to Tammy and the RPOs to make sure we got the messaging right before we publish these. Then turn to identifying whether there are any other places to mention QuickSubmit or change the messaging from AVA to QuickSubmit.

**Comment by katherine-fung** [comment]
Hi @megzehn, can you please copyedit and publish these changes on Monday?

How to apply for the GI Bill
http://preview-prod.vfs.va.gov/preview?nodeId=943 

Get paid back for test fees
http://preview-prod.vfs.va.gov/preview?nodeId=926 

Thank you!

**Comment by megzehn** [comment]
Thank you, @katherine-fung! I published these 2 pages. 

Noting that there's a broken link in Get paid back for test fees: https://inquiry.vba.va.gov/weamspub/buildSearchCountryLCCriteria.do

Thank you for asking the SMEs about it. When they respond, I can either update the link or open a new ticket to adjust the text.

**Comment by katherine-fung** [comment]
Hi @megzehn - Tammy let us know that the broken link is working again. I can close this ticket.
